src/frontend/src/tests/mocks/page.store.mock.ts

```diff
@@ -1,13 +1,30 @@
+import { NETWORK_PARAM, TOKEN_PARAM, URI_PARAM } from '$lib/constants/routes.constants';
+import type { Page } from '@sveltejs/kit';
 import { writable } from 'svelte/store';
 
+export const mockPageData = ({
```

Comment:
In NNS dapp we replace the page with a store for test purpose that exposes a `mock` function. That way in the test you can call `page.mock`.

Would that be something interesting?

https://github.com/dfinity/nns-dapp/blob/95d58213187190f606cffe7b43e6fc857d50528d/frontend/__mocks__/%24app/stores.ts#L40

Reply:
Yes! Good suggestion! Done:

- PR #3415